CMakeLists.txt

```diff
@@ -117,7 +117,7 @@ if(CCTAG_WITH_CUDA)
     set(CUDA_NVCC_FLAGS_DEBUG   "${CUDA_NVCC_FLAGS_DEBUG};-G;-g")
   else()
     message(STATUS "Building in release mode")
-    add_definitions(-DNDEBUG)
+    add_compile_definitions(NDEBUG)
```

Comment:
CMake automatically add it to the flags if the build is release, no need to specify it

You can check here, this is the branch before you added it explicitly
https://travis-ci.org/alicevision/CCTag/jobs/563273927#L4118
and this is now
https://travis-ci.org/alicevision/CCTag/jobs/563423199#L4118
(it is set twice)

Reply:
Hmm. It appears to be added correctly when CMAKE_BUILD_TYPE is explicitly set to "Release" (which Travis is doing). If the CMAKE_BUILD_TYPE is left undefined, neither "Release" nor "Debug" flags are added.
I suppose that the better solution would be force a default CMAKE_BUILD_TYPE of either "Release" or "Debug"?